notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb

```diff
@@ -0,0 +1,1586 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                                        pyarrow -q
</pre><p>Do you use pyarrow in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yeah, It's used for downloading BigQuery tables to DataFrame. It's one of the dependencies and without installing it manually it will end up in error!